cfme/infrastructure/host.py

```diff
@@ -214,6 +217,12 @@ def delete(self, cancel=True):
         cfg_btn(btn_name, invokes_alert=True)
         sel.handle_alert(cancel=cancel)
 
+    def load_details(self, refresh=False):
+        """To be compatible with the Taggable and PolicyProfileAssignable mixins."""
+        navigate_to(self, 'Details')
```

Comment:
Thinking of adding a refresh option to all pages, like the use_resetter.....thoughts? @mfalesni @RonnyPfannschmidt @jkrocil @izapolsk @mshriver @otsuman